db/migrate/20240427023227_create_employee_teams.rb

```diff
@@ -3,6 +3,8 @@ def change
     create_table :employee_teams do |t|
       t.date :start_date
       t.date :end_date
+      t.references :employee, null: false
```

Comment:
Lines 6 and 7 here are actually not needed, since those two columns are added in a later migration: 0240430003654_add_employee_and_team_to_employee_teams.rb